src/net_types.h

```diff
@@ -6,10 +6,55 @@
 #define BITCOIN_NET_TYPES_H
 
 #include <map>
+#include <serialize.h>
```

Comment:
why?

Reply:
> why?

I must have added it to avoid compile errors before #22849. Probably wasn't the right place to add the include in any case, though. Will remove